rero/rero-ils
# SRU: implement CQL search and the explain response

* Adds CQL parser.
* Adds explaine response.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- Implements SRU CQL search and explain.

## How to test?

See the explain page:
https://localhost:5000/api/sru/documents
Try to use the SRU search:
https://localhost:5000/api/sru/documents?version=1.1&operation=searchRetrieve&query=dc.title=Jonni
